docs/troubleshooting-guide/saml.md

```diff
@@ -31,6 +35,10 @@ After filling out the authentication form with your identity provider, you go ba
 1. You should have received an XML file from your identity provider. Open that metadata file, and check to make sure the certificate you inputted is correct.
 2. Go to **Admin settings** > **Settings** > **Authentication** > **SAML**. Check that the certificate that you entered into the **SAML Identity Provider Certificate** field matches the certificate in the XML file you got from your identity provider. Depending on your provider, you might need to download this, open it in a text editor, then copy and paste the certificate's contents into the field.
 
+## Is the SSO URL correct?
+
+Verify that the Single Sign On URL (or equivalent) that you enter on your SAML provider’s website has `/auth/sso` appended to it. For instance, if you want your users to end up at `https://metabase.mycompany.com`, the full URL should be `https://metabase.mycompany.com/auth/sso`.
```

Comment:
Good example here.

Reply:
I moved this paragraph from the original SAML doc, so credit for this example probably goes back to you ;)